libs/parallelism/executors/include/hpx/executors/detail/hierarchical_spawning.hpp

```diff
@@ -56,6 +55,9 @@ namespace hpx { namespace parallel { namespace execution { namespace detail {
         std::size_t const size = hpx::util::size(shape);
         results.resize(size);
 
+        auto post_policy = policy;
+        post_policy.set_stacksize(threads::thread_stacksize::small_);
```

Comment:
Do you count this as "internal use", hence `set_stacksize` and not `with_stacksize` (if yes, :+1:)?

Reply:
Yes, internally I have used the `set_` versions, everywhere.